config/routes.rb

```diff
@@ -148,7 +148,7 @@
 
     resources :correspondence, controller: "investigations/correspondence_routing", only: %i[new create]
     resources :emails, controller: "investigations/record_emails", only: %i[new create]
-    resources :phone_calls, controller: "investigations/record_phone_calls", only: %i[show new create update]
+    resources :phone_calls, controller: "investigations/record_phone_calls", only: %i[new create], path: "record-phone-calls"
```

Comment:
I think we could match the path to the one used by the `show` action (defined separately above), with the intention that we'll soon be able to merge those controllers, and add edit/update actions?

```suggestion
    resources :phone_calls, controller: "investigations/record_phone_calls", only: %i[new create], path: "phone-calls"
```